elfo-core/src/logging/filter.rs

```diff
@@ -1,55 +1,32 @@
-use tracing::{level_filters::LevelFilter, Metadata, Subscriber};
-use tracing_subscriber::{filter::Targets, layer::Context, Layer};
+use tracing::{level_filters::LevelFilter, Metadata};
```

Comment:
The whole `LogFilter` can be removed now because enabled levels are already checked by
```rust
            if !scope.permissions().is_logging_enabled(level) {
                return false;
            }
```
in the filtering layer.